pkg/helm/release/manager.go

```diff
@@ -308,5 +329,10 @@ func (m manager) UninstallRelease(ctx context.Context) (*rpb.Release, error) {
 
 	uninstall := action.NewUninstall(m.actionConfig)
 	uninstallResponse, err := uninstall.Run(m.releaseName)
+	for _, o := range opts {
+		if err := o(uninstall); err != nil {
+			return nil, fmt.Errorf("failed to apply uninstall option: %w", err)
+		}
+	}
```

Comment:
This needs be moved before the call to `uninstall.Run()`

Reply:
ups